Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerDownloadTask.cpp

```diff
@@ -56,7 +56,7 @@ ServiceWorkerDownloadTask::ServiceWorkerDownloadTask(NetworkSession& session, Ne
     , m_serverConnectionIdentifier(serverConnectionIdentifier)
     , m_fetchIdentifier(fetchIdentifier)
     , m_downloadID(downloadID)
-    , m_networkProcess(serviceWorkerConnection.networkProcess())
+    , m_networkProcess(serviceWorkerConnection.protectedNetworkProcess().releaseNonNull())
```

Comment:
This is 100% equivalent, I changed back to legacy coding style

Reply:
To be clear, the coding style says to use `protectedFoo()` functions only to avoid local variables. It is never meant to be used for variable / data member initialization. As such `*serviceWorkerConnection.networkProcess()` is still the recommended coding style.

You - yourself - said you didn't like the `protectedFoo()` calls because they're longer. Therefore, I'm surprised you'd want to use them in cases where they don't help at all.